.github/workflows/release-docker.yml

```diff
@@ -0,0 +1,23 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: release-docker
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release-docker:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Release docker image
+        uses: reanahub/reana-github-actions/release-docker@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          organisation: reanahub
```

Comment:
Two minor musings, we can fully forget them:

- Perhaps "organisation" could also be a secret so that the setup could be all secretified for a possible theoretical replication elsewhere.

- I guess we shell be using access tokens, so perhaps we could even name the PASSWORD in a ACCESS_TOKEN style. (Even though the tokens work in place of passwords, so the naming is already OK in this way; just thinking out loud about possible "discouraging" of the use of passwords here via a visible name.)

Reply:
I have added `vars.DOCKER_ORGANISATION` and ` vars.DOCKER_REGISTRY`. I am also using the boolean `vars.RELEASE_DOCKER` so that this job does not run on our own forks